insights/parsers/certificates_enddate.py

```diff
@@ -42,13 +42,14 @@
 from insights.specs import Specs
 
 
+# namedtuple: contains the expiration date in string and datetime format.
+ExpirationDate = namedtuple('ExpirationDate', ['str', 'datetime'])
```

Comment:
Actually, this change modifies the API of the Parser.  However, it makes sense for the new parser and doesn't break any existing rule. It looks acceptable to me. How about you @bfahr ?

Reply:
@xiangce @huali027 this change isn't really necessary and it does impact anyone that is using this parser and the `ExpirationDate` tuple.  I think we should stick to our rule to maintain backward compatibility and not make this change.